EstateAgency.Tests/EstateAgencyTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿using EstateAgency.Domain;
+using EstateAgency.Domain.Enum;
+using EstateAgency.Domain.Data;
+using Newtonsoft.Json.Linq;
+
+namespace EstateAgency.Tests;
+
+/// <summary>
+/// Содержит тесты для проверки запросов к исходным данным о недвижимости
+/// </summary>
+public class EstateAgencyTests
+{
+    private readonly List<Client> _clients;
+    private readonly List<Property> _properties;
+    private readonly List<Request> _requests;
+
+    /// <summary>
+    /// Загружает тестовые данные из DataSeeder и устанавливает связи между объектами
+    /// </summary>
+    public EstateAgencyTests()
+    {
+        var testData = DataSeeder.GetCompleteTestData();
+        _clients = testData.clients;
+        _properties = testData.properties;
+        _requests = testData.requests;
```

Comment:
Я бы упростил путем деконструкции картежа:
```suggestion
        (_clients, _properties, _requests) = DataSeeder.GetCompleteTestData();
```